src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -2349,64 +2300,13 @@ defaultImports icUsesGrpc | StringType stringModule stringType <- ?stringType =
     , importDecl_ (m "Network.GRPC.HighLevel.Server")              & alias grpcNS & hiding    [i"serverLoop"]
     , importDecl_ (m "Network.GRPC.HighLevel.Server.Unregistered") & alias grpcNS & selecting [i"serverLoop"]
     ])
-    <>
-    ( if not ?typeLevelFormat then [] else
-    [ importDecl_ (m "Proto3.Suite.Form") & qualified protobufFormNS & everything
-    , importDecl_ (m "GHC.TypeLits")      & qualified haskellNS      & selecting [i"Nat", i"Symbol", i"TypeError"]
-    ])
```

Comment:
Please restore the support for type-level format information; I think it got deleted during merge.